src/xdocs/checks/javadoc/javadocpackage.xml

```diff
@@ -58,15 +58,77 @@
   &lt;module name=&quot;JavadocPackage&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p id="Example1-structure">
+          File structure with no package-info.java file in directory:<br/><br/>
```

Comment:
Directory structure .....


Fix in all cases

Reply:
Done